README.rst

```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
 
    http://statsmodels.github.io/dev/index.html
 
-
-This is currently a temporary solution that will be transformed into the new permanent location.
+**Note:** This is currently a temporary solution that will be transformed into the new permanent location.
 
 
 Main Features
 =============
 
-* linear regression models: Generalized least squares (including weighted least squares and
-  least squares with autoregressive errors), ordinary least squares.
-* glm: Generalized linear models with support for all of the one-parameter
+* Linear regression models:
+
+  - Ordinary least squares
+  - Generalized least squares
+  - Weighted least squares
+  - Least squares with autoregressive errors
+
+* GLM: Generalized linear models with support for all of the one-parameter
   exponential family distributions.
```

Comment:
missing (after GLM)
`* GEE: Generalized Estimating Equations for one-way clustered or longitudinal data.`